### PR TITLE
Add more HTML e-mails

### PR DESCRIPTION
**Pull request by djahnie**
I made mjml files for all the templates shown in the nice template viewer.
Just some issues:

- `accountdelete_request` and `changemail_success` can never have worked, can it? I mean, there is no url specified in `email_utils.py`. I'm not really sure how to add them there either, so I didn't for now.
- Which way of password reset is actuall in use? `passwordreset_request` already offers a link to change the password (even though it's broken/unspecified), so why or when do we need `newpassword`? Is any of these mails an unused legacy thing or am I missing something here?

### CODE REVIEW

File: foodsaving/users/templates/mailverification.mjml [hunk 1]
```diff
@@ -13,7 +13,7 @@
                     </mj-text>
                     <mj-text>
                         <p>{% trans %}Welcome to {{ site_name }}, {{ user_display_name }}!{% endtrans %}</p>
-                        <p>{% trans %}After we get started, please confirm your email address by clicking the link below:{% endtrans %}</p>
+                        <p>{% trans %}To get started, please confirm your email address by clicking the link below:{% endtrans %}</p>
```
[the remark this answers is not in the document]
Comment (djahnie) [reply foodsaving/users/templates/mailverification.mjml]: Hmmmm... I get your point but it's just a really strange phrasing...
I'm torn on this one... :confused:

File: foodsaving/users/templates/accountdelete_request.mjml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <mj-section>
+                <mj-column>
+                    <mj-text>
+                        <h1>{% trans %}You requested to delete your account on {{ site_name }}!{% endtrans %}</h1>
+                    </mj-text>
+                    <mj-text>
+                        <p>{% trans %}We're sad to hear that you want to delete your account, could we have done anything better?{% endtrans %}</p>
+                        <p>{% trans %}Anyhow, you can finish the deletion by clicking the link below:{% endtrans %}</p>
+                    </mj-text>
+                    <mj-button href="{{ url }}">
```
[the remark this answers is not in the document]
Comment (djahnie) [reply foodsaving/users/templates/accountdelete_request.mjml]: Hehe true! It's the final button after all! ^^

File: foodsaving/users/templates/mailverification.mjml (hunk 1 above)
[the remark this answers is not in the document]
Comment (djahnie) [reply foodsaving/users/templates/mailverification.mjml]: Argh! Okay translation is more important than fitting, logical sentences... :pensive:

**Comment by tiltec** [comment]
Three tests are failing, so this has to wait a bit more

**Comment by tiltec** [comment]
@mr-kojo should I merge this into #501?

**Comment by mr-kojo** [comment]
@tiltec yes, thanks!

**Pull request merged**